src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
@vkurchatkin unhandled rejections are supposed to be _very_ rare - there should not be more than 10 of those per hour tops. You're opting for killing the app if a default handler isn't attached iirc.

If you think this is important then I'll take your word since you have a lot more io.js experience than me with these things but I don't see the big merit of optimizing this.